docs/api/profiles.md

```diff
@@ -0,0 +1,123 @@
+# Profiles
+
+Profiles contain information about a User. A User does not need a Profile.
+
+## Create a profile
+
+Link a profile to a user. If a user already has a profile, it will be overwritten.
+
+### URL
+
+```http
+POST /users/:userId/profile HTTP/1.1
+```
+
+### Input
+
+Name         | Type       | Description
+-------------|------------|------------
+birthYear    | `int`      | **Optional** The user's year of birth.
+gender       | `string`   | **Optional** The user's gender ("Male"|"Female").
+country      | `string`   | **Optional** The country of the user's mother tongue.
+language     | `string`   | **Optional** The language of the user's mother tongue. In ISO 639-2 format.
+cefr         | `string`   | **Optional** The user's CEFR (Common European Framework of Reference) level.
```

Comment:
The PR is fine as is. Perhaps it's good to link or show the actual cefr table as I saw it in the its-gae issue?

Reply:
Well the table in the its-gae issue, comes from the https://www.english.com/gse/teacher-toolkit/user/lo, I'm not sure if that's an official standard. However the CEFR of course is, this site https://www.cambridgeenglish.org/exams-and-tests/cefr/ best explains the standard. However providing the complete table such as outlined on that site would be a bit overkill.

Reply:
Perhaps a link to the pdf? https://www.cambridgeenglish.org/Images/126130-cefr-diagram.pdf

Reply:
of course, linking is fine.

Reply:
The scale on the issue (< A1, A1, A2, A2+, B1, B1+, B2, B2+, C1, C2) is the "official" CEFR scale.. The table in the description was just a way to store it in the backend, so yes, I would still prefer the cefr to be an int from 0 to 10, which translates to the string values...

Here is the table again:

| int | value    | 
| --- | -------- |
| 0   | no score |
| 1   | < A1     |
| 2   |   A1     |
| 3   |   A2     |
| 4   |   A2+    |
| 5   |   B1     |
| 6   |   B1+    |
| 7   |   B2     |
| 8   |   B2+    |
| 9   |   C1     |
| 10  |   C2     |

The table in the comments which refers to the english.com
https://www.english.com/gse/teacher-toolkit/user/lo website regarding GSE is also official, it maps the GSE (used for English) to CEFR.

And that is the point: CEFR can be used for most languages we would like to use (Dutch, German, English, French..).